integration_tests/snaps/simple-svn/snapcraft.yaml

```diff
@@ -0,0 +1,11 @@
+name: test-package
+version: 0.1
+summary: svn integration test
+description: use svn
+confinement: strict
+
+parts:
+    svn:
+        plugin: make
+        source: .
+        source-type: svn
```

Comment:
@tsimonq2, yes, I mean that the source-type couldn't have matched anything in the test, earlier, and should have caused a test failure.